gen-compdocs/generators/doc.go

```diff
@@ -85,10 +96,13 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 	// Note: Files generated for kubeadm tool are snippets of Markdown without a title.
 	// These snippets are included in the corresponding kubeadm pages.
 	if withTitle {
-		if _, err := fmt.Fprintf(w, "---\ntitle: %s\ncontent_type: tool-reference\nweight: 30\n---\n\n", name); err != nil {
+		if _, err := fmt.Fprintf(w, "---\ntitle: %s\ncontent_type: tool-reference\nweight: 30\ndoc_type: generated\n---\n\n", name); err != nil {
 			return err
 		}
 
+		// Print the "generated" warning
+		fmt.Fprintf(w, "%s\n\n", generated_warning)
```

Comment:
This warning appears immediately after the title as a comment.
nit: Instead of introducing `doc_type`, what do you think about adding a boolean? Something like: `auto-generated: true`?

Reply:
Yes. I'm afraid putting the warning at the very head of the page may disrupt Docsy.
Boolean tag is a good idea.
Once we have this tag, we can introduce some intelligence (as you suggested before) into Hugo so that the generated page doesn't have those 'edit' buttons.